server/svix-server/src/core/cache/memory.rs

```diff
@@ -8,21 +8,24 @@ use tokio::{
 };
 
 use axum::async_trait;
-use serde_json;
 use std::collections::HashMap;
 use std::sync::Arc;
 
-use super::{Cache, CacheBehavior, CacheKey, CacheValue, Result};
+use super::{Cache, CacheBehavior, CacheKey, Result};
+
+// Apparently traits needed by types in default implementation aren't recognized:
+#[allow(unused_imports)]
+use super::{CacheValue, StringCacheValue};
```

Comment:
This doesn't make sense... I'm sure it was something else and removing this will work.